### ISSUE TITLE
Code Llama 13b download failed: no properly formatted checksum lines found

### ISSUE THREAD

**Issue by kraso** [meta-llama/codellama#197]
I ran the bash script from the command console and received the following message and the download was aborted:

Checking checksums
md5sum: checklist.chk: no properly formatted checksum lines found

Any ideas?

**Comment by ankitrs7** [comment]
Got same issue.

**Comment by jgehring** [comment]
Can you double-check the contents of the `checksum.chk` in the directory that was downloaded?

**Comment by kraso** [comment]
> Can you double-check the contents of the `checksum.chk` in the directory that was downloaded?

Sorry, i had to format my computer again and i lost these files. I will try to install it again

**Comment by kraso** [comment]
> Can you double-check the contents of the `checksum.chk` in the directory that was downloaded?

[checklist.zip](https://github.com/facebookresearch/codellama/files/14150812/checklist.zip)

Hello, here is a compressed copy of the file.

**Comment by jgehring** [comment]
Thanks for sharing; your checklist file seems to be an HTML page for signing into a Google account. Just to double-check, you ran `bash download.sh` and pasted the link you received in the e-mail from Meta, titled "Get started with Code Llama"? And this operation produced this checksum.chk file? What do the other files in the directory where you have the checksum file look like (name and size)?

**Comment by kraso** [comment]
> Thanks for sharing; your checklist file seems to be an HTML page for signing into a Google account. Just to double-check, you ran `bash download.sh` and pasted the link you received in the e-mail from Meta, titled "Get started with Code Llama"? And this operation produced this checksum.chk file? What do the other files in the directory where you have the checksum file look like (name and size)?

No, I didn't have time to paste the link that I received in the email because no interface appeared to provide that information. Unless you have to supply the information to the command as an argument, I don't know how to pass the email URL to it.

These are the files into the CodeLlama-13b directory:

-rw-r--r-- 1  172576 feb  3 09:56 checklist.chk
-rw-r--r-- 1  172177 feb  3 09:56 consolidated.00.pth
-rw-r--r-- 1  172167 feb  3 09:56 consolidated.01.pth
-rw-r--r-- 1  172152 feb  3 09:56 params.json
-rw-r--r-- 1  172013 feb  3 09:56 tokenizer.model

**Comment by allelive** [comment]
> > Grazie per la condivisione; il file della tua lista di controllo sembra essere una pagina HTML per l'accesso a un account Google. Giusto per ricontrollare, hai eseguito `bash download.sh`e incollato il collegamento ricevuto nell'e-mail da Meta, intitolato "Inizia con Code Llama"? E questa operazione ha prodotto questo file checksum.chk? Che aspetto hanno gli altri file nella directory in cui hai il file di checksum (nome e dimensione)?
> 
> No, non ho avuto il tempo di incollare il collegamento che ho ricevuto nell'e-mail perché nessuna interfaccia sembrava fornire tali informazioni. A meno che tu non debba fornire le informazioni al comando come argomento, non so come passargli l'URL dell'e-mail.
> 
> Questi sono i file nella directory CodeLlama-13b:
> 
> -rw-r--r-- 1 172576 3 feb 09:56 checklist.chk -rw-r--r-- 1 172177 3 feb 09:56 consolidato.00.pth -rw-r--r-- 1 172167 3 feb 09:56 consolidato.01.pth -rw-r--r-- 1 172152 3 feb 09:56 params.json -rw-r--r-- 1 172013 3 feb 09:56 tokenizer.model

pip install -r requirements.txt
./download.sh quando dai questo comando ti chiede di inserire il link ricevuto in email. se da problemi controlla gli spazi che non devono esserci nel link oppure trascrivilo manualmente se non parte il download e solo problema del link che quando tu lo copi e poi incolli ti modifica il link altrimenti elimini la email e richiedi un link nuovo ma se richiedi il link nuovo vai nella directory codellama e cancella le versioni dei modelli che hai tentato di scaricare e riprovi con il nuovo link controllando che non ci siano spazi in pratica deve essere tutto blu il link spero di averer spiegato bene io cosi ho risolto buon fine settimana.. con la cli lo puoi usare nel terminale e in pratica uhaoooo

**Comment by kraso** [comment]
I'm very sorry, I had read the instructions very wrong and when I was asked for the email url, I simply indicated my email account instead of the URL.

I have deleted the folder with all the previous configurations and I have downloaded the installation file again from the following link:

[https://github.com/facebookresearch/codellama/archive/refs/heads/main.zip](https://github.com/facebookresearch/codellama/archive/refs/heads/main.zip)

I have unzipped and launched the bash script with the following command:

`bash download.sh`

It asks me for the URL and I enter it. It asks me which model I want to install and, in my case, I enter 13b. Press enter and the download begins. Finally, it seems that the download of the files and their verification has been correct and everything has gone well.

Now I have the installation done but I don't know the steps to follow to be able to interact with the model. I am going to continue reading the instructions and try to follow them to the letter.

I have learned that it is important to read things twice before making rash decisions.

Thank you very much for your time.
